corehq/apps/locations/bulk_management.py

```diff
@@ -539,7 +539,6 @@ def errors(self):
 
         return errors
 
-    @memoized
```

Comment:
curious how you determined these were unneeded? is it because they don't do any heavy work or because they are only ever called once (and presumably guaranteed to not be later changed to be called more than once)?

Reply:
They were (and still are) only ever called once. Previously the thing that called them was also `@memoized`, and now they're called during `__init__` (and they're private), so not likely to be called multiple times in the future.

Reply:
roger, thanks. i started wondering if we would ever get value from something like an `assert_called_once` decorator that just checks a counter against function calls and fails if it's ever > 1. and then i decided that was almost certainly overkill in all scenarios i could think of.